components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/config/builder/FileBasedConfigurationBuilder.java

```diff
@@ -197,13 +199,17 @@ private void buildConfiguration(InputStream inStream) throws IOException {
 
             //########### Read Sequence Configs ###########
             readSequenceConfigs(rootElement);
+
+            //########### Read JIT Provisioning Configs ##########
```

Comment:
Any particular reason for the comment? Otherwise, method name is intuitive enough.